### ISSUE TITLE
Titlebar can be hidden by the global menubar

### ISSUE THREAD

**Issue by primis** [AgoraDesktop/apps-gworkspace#9]
With the addition of the global menubar, it is now possible to drag a window such that it's titlebar is completely covered by the menubar. This makes it impossible to move the window in question.

**Comment by KyleCardoza** [comment]
Yes, this is absolutely a thing, and absolutely undesirable behaviour. This will require adjusting the window manager such that it refuses to move a window to cover that bar.

**Comment by KyleCardoza** [comment]
I have determined that, when a given window is an AppKit application window, the window decorations of which are drawn by AppKit (which is the intended use case in Agora, plain X11 applications are officially unsupported), the window cannot be dragged over or under the global menubar.

That being said, when we determine the final X11 window manager, this behaviour with plain X11 windows will be addressed.
